<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BB_EasynForm</class>
 <widget class="QWidget" name="BB_EasynForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frameEasyn{
	
	background-color: rgb(167, 167, 167);
}
QFrame#frameEasyn QPushButton{
	border:none;
	color: rgb(255, 255, 255);
	background-color: rgb(182, 182, 182);
}

QFrame#frameEasyn QPushButton:hover{
	border:none;
	color: rgb(255, 255, 255);
	background-color: rgb(162, 162, 162);
}
QFrame#frameEasyn QPushButton:pressed{
	border:none;
	color: rgb(255, 255, 255);
	background-color: rgb(142, 142, 142);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameEasyn">
     <property name="styleSheet">
      <string notr="true">QFrame#frameEasyn{
	
	background-color: rgb(67, 66, 71);
}
QPushButton{
	border:none;
	color: rgb(255, 255, 255);
	font: 9pt &quot;微软雅黑&quot;;
	background-color: rgb(87, 86, 91);
}


QPushButton:hover{
	border:none;
	color: rgb(255, 255, 255);
	font: 9pt &quot;微软雅黑&quot;;
	background-color: rgb(107, 106, 101);
}
QPushButton:pressed{
	border:none;
	color: rgb(255, 255, 255);
	font: 9pt &quot;微软雅黑&quot;;
	background-color: rgb(59, 58, 63);
}
QPushButton:disabled{
	border:none;
	color: rgb(255, 255, 255);
	font: 9pt &quot;微软雅黑&quot;;
	background-color: rgb(130, 130, 130);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,2,2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnPannel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>操作台 &gt;&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPre">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>上一步</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnSlice">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>切割</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDraw">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>雕刻</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/Source/icon/print_begin.png</normaloff>:/Source/icon/print_begin.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/Source/icon/print_cancle.png</normaloff>:/Source/icon/print_cancle.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
